pom.xml

```diff
@@ -4,7 +4,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.github.hi-fi</groupId>
 	<artifactId>httpclient</artifactId>
-	<version>0.0.6-SNAPSHOT</version>
+	<version>0.0.7-SNAPSHOT</version>

```

Comment:
No need to increase the version. Last release was 0.0.5, so next is going to be 0.0.6, not 0.0.7.

Reply:
> Have you tested this locally with https://github.com/Hi-Fi/robotframework-httprequestlibrary? Is this still working as should with it, and provide the information you were needing?

I've just tested locally. It works good for me.